Themes/default/languages/ModerationCenter.english.php

```diff
@@ -172,6 +172,7 @@
 
 // Do not use numeric entries in the below string.
 $txt['mc_reportedm_ignore_confirm'] = 'Are you sure you wish to ignore further reports about this user\'s profile?\\n\\nThis will turn off further reports for everyone.';
+$txt['mc_reportedm_ignore_confirm'] = 'If you choose &quot;OK&quot;, all future reports about this user\'s profile will be ignored; moderators will not see any future reports about this user\'s profile . Are you sure you wish to do this?';
```

Comment:
duplicate key?

Reply:
Addressed in https://github.com/SimpleMachines/SMF2.1/pull/5778/commits/b68a6dcdc21ea1244e9011f7d62c7b20b3b54c50